src/components/accessories/dateField/DateField.tsx

```diff
@@ -45,10 +45,12 @@ const DateField: FunctionComponent<IProps> = ({
         format={format}
         id={fieldName}
         name={fieldName}
-        label={required === FIELD_VALIDATION.SUGGESTED ? label + " **" : label}
+        label={label}
```

Comment:
@axelpezzo I'm wondering if the code removed here was your work on that "suggested-filling fields" feature. Is this change ok?

Reply:
Yep, we need to restore this. @SteveGT96